tests/api/v1/tokens/revokeRestore.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/api/v1/tokens/revokeRestore.js
+ */
+'use strict';
+
+const supertest = require('supertest');
+const api = supertest(require('../../../../index').app);
+const constants = require('../../../../api/v1/constants');
+const tu = require('../../../testUtils');
+const u = require('./utils');
+const path = '/v1/tokens';
+const expect = require('chai').expect;
+const Profile = tu.db.Profile;
+const User = tu.db.User;
+const Token = tu.db.Token;
+
+describe(`api: POST ${path}/:id/revoke and POST ${path}/:id/restore`, () => {
+  let usr;
+  let tid;
+
+  beforeEach((done) => {
+    Profile.create({
+      name: `${tu.namePrefix}testProfile`,
+    })
+    .then((profile) =>
+      User.create({
+        profileId: profile.id,
+        name: `${tu.namePrefix}test@refocus.com`,
+        email: `${tu.namePrefix}test@refocus.com`,
+        password: 'user123password',
+      })
+    )
+    .then((user) => {
+      usr = user;
+      return Token.create({
```

Comment:
@iamigo can do have a test where we expect the default value of token.isRevoked === 0?